### DISCUSSION TITLE
shader backup script

### DISCUSSION THREAD

**Discussion by thatguyjk** [Gargaj/Bonzomatic#157] [Q&A]
Does anyone have a bash (.sh) script that would do the same thing as the windows batch script to backup a shader on close?

**Comment by PoroCYon** [comment]
idk, something like this?

```sh
cp "$1" "path/to/backupfolder/shader-$(date +%s).glsl"
```

(didn't test this at all)

**Reply by thatguyjk** [reply]
Thank you for the quick response, I'll give it a try.

**Reply by thatguyjk** [reply]
Going to open an issue, but it doesn't look like the postExitCmd is being executed.
